### PR TITLE
Improve ActionDetails.md.erb look at actual file names

### LINKED ISSUE #12107: ActionDetails.md.erb needs to dynamically update action path in comment for updating
This line is wrong when it generates the new doc for an action - https://github.com/fastlane/fastlane/blob/master/fastlane/lib/assets/ActionDetails.md.erb#L3

This should generate something like...
```
To modify it, go to its source at https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/s3.rb
```

### PR DESCRIPTION
**Pull request by joshdholtz**
Fixes #12107

- comment links directly to file that action is in for editing (previously linked to ActionDetails.md.erb
- source link to actual file action is in (assume file name by action name)
- doesn’t generate actions that were actually plugins which fastlane uses for build/dev stuff

#### Comment will now link to action file for changing
```
To modify it, go to its source at https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/s3.rb
```

#### Source file will now link to proper source location in GitHub 
https://docs.fastlane.tools/actions/xcbuild/
before: https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/xcbuild.rb
now: https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/xcodebuild.rb

### Was loading plugins that fastlane used for building 
- https://docs.fastlane.tools/actions/clubmate
- https://docs.fastlane.tools/actions/plugin_scores
- https://docs.fastlane.tools/actions/rspec
- https://docs.fastlane.tools/actions/rubocop
- https://docs.fastlane.tools/actions/slack_train_action
- https://docs.fastlane.tools/actions/slack_train_crash_action
- https://docs.fastlane.tools/actions/slack_train_start_action

and it now will skip these

```sh
ERROR [2018-03-21 09:09:43.73]: Action 'Fastlane::Actions::ClubmateAction' not found in root fastlane project... skipping
DEBUG [2018-03-21 09:09:43.73]: Action 'Fastlane::Actions::ClubmateAction' found at /Users/josh/Projects/fastlane/fastlane/fastlane/lib/fastlane/actions/clubmate_action.rb
ERROR [2018-03-21 09:09:44.72]: Action 'Fastlane::Actions::PluginScoresAction' not found in root fastlane project... skipping
DEBUG [2018-03-21 09:09:44.72]: Action 'Fastlane::Actions::PluginScoresAction' found at /Users/josh/Projects/fastlane/fastlane/fastlane/lib/fastlane/actions/plugin_scores.rb
ERROR [2018-03-21 09:09:44.81]: Action 'Fastlane::Actions::RspecAction' not found in root fastlane project... skipping
DEBUG [2018-03-21 09:09:44.81]: Action 'Fastlane::Actions::RspecAction' found at /Users/josh/Projects/fastlane/fastlane/fastlane/lib/fastlane/actions/rspec.rb
ERROR [2018-03-21 09:09:44.82]: Action 'Fastlane::Actions::RubocopAction' not found in root fastlane project... skipping
DEBUG [2018-03-21 09:09:44.82]: Action 'Fastlane::Actions::RubocopAction' found at /Users/josh/Projects/fastlane/fastlane/fastlane/lib/fastlane/actions/rubocop.rb
ERROR [2018-03-21 09:09:44.89]: Action 'Fastlane::Actions::SlackTrainAction' not found in root fastlane project... skipping
DEBUG [2018-03-21 09:09:44.89]: Action 'Fastlane::Actions::SlackTrainAction' found at /Users/josh/Projects/fastlane/fastlane/fastlane/lib/fastlane/actions/slack_train_action.rb
ERROR [2018-03-21 09:09:44.89]: Action 'Fastlane::Actions::SlackTrainCrashAction' not found in root fastlane project... skipping
DEBUG [2018-03-21 09:09:44.89]: Action 'Fastlane::Actions::SlackTrainCrashAction' found at /Users/josh/Projects/fastlane/fastlane/fastlane/lib/fastlane/actions/slack_train_crash_action.rb
ERROR [2018-03-21 09:09:44.90]: Action 'Fastlane::Actions::SlackTrainStartAction' not found in root fastlane project... skipping
DEBUG [2018-03-21 09:09:44.90]: Action 'Fastlane::Actions::SlackTrainStartAction' found at /Users/josh/Projects/fastlane/fastlane/fastlane/lib/fastlane/actions/slack_train_start_action.rb
```

### CODE REVIEW

File: fastlane/lib/fastlane/documentation/markdown_docs_generator.rb [hunk 1]
```diff
@@ -43,6 +43,40 @@ def all_actions_from_enhancer
       @_launches ||= JSON.parse(File.read(File.join(Fastlane::ROOT, "assets/action_ranking.json"))) # root because we're in a temporary directory here
     end
 
+    def actions_path
+      "lib/fastlane/actions/"
+    end
+
+    def where_is(klass)
```
Comment (janpio) [inline fastlane/lib/fastlane/documentation/markdown_docs_generator.rb]: what does `klass` stand for?

Comment (joshdholtz) [reply fastlane/lib/fastlane/documentation/markdown_docs_generator.rb]: Its the `class` of the action but `class` is a reserved word so we have to go with `klass` 😇

Comment (janpio) [reply fastlane/lib/fastlane/documentation/markdown_docs_generator.rb]: omg

**Review by revolter — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**